src/app/submissions/submissions.component.ts

```diff
@@ -16,7 +16,7 @@ import { DialogsLoadingService } from '../shared/dialogs/dialogs-loading.service
     .mat-column-name {
       max-width: 25vw;
     }
-    mat-row {
+    .mat-row-pointer {
```

Comment:
After converting to use to the global class you can remove this definition entirely.